matlab/lib/+Ice/createProperties.m

```diff
@@ -1,45 +1,36 @@
-function [properties, remArgs] = createProperties(varargin)
-    % createProperties  Creates a property set, optionally initialized from an
-    %   argument vector and default properties.
+function [properties, remArgs] = createProperties(args, defaults)
+    % createProperties  Creates a property set, optionally initialized from an argument vector and default properties.
     %
     % Examples:
     %   props = Ice.createProperties();
     %   [props, remArgs] = Ice.createProperties(args);
-    %   props = Ice.createProperties(defaults);
```

Comment:
While documented, this option was actually not implemented. This way, I could remove it and use an arguments block.